cypress/integration/expand-spec.js

```diff
@@ -0,0 +1,23 @@
+import { Preview } from '../page-objects/preview-page'
+
+const preview = new Preview()
+
+describe( 'Check the Expanded Layout', () => {
+
+	beforeEach( 'Open the English Page', () => {
+		cy.navigateToTestPage()
+	} )
+
+	it( 'Check the Wikipedia Preview in Expanded Layout', () => {
+		// Opens the Preview
+		preview.getPreviewSpan().eq( 0 ).click()
+		// Click on Continue Reading Footer element
+		preview.getFooterContiReading().click()
+		// Check the Expanded Mode preview
+		preview.checkPreviewExpanded()
+		// Closes the Preview
+		preview.getHeaderClosebtn().click()
```

Comment:
why do u need this click after the checks?

Reply:
It basically checks whether the preview is being opened, by actually closing it. Because the 'checks' only check the visibility of elements only. So we need to make some clicks to assert our case.